integreat_cms/cms/constants/roles.py

```diff
@@ -73,16 +73,19 @@
 MANAGEMENT_PERMISSIONS = EDITOR_PERMISSIONS + [
     "change_feedback",
     "change_imprintpage",
+    "change_organization",
```

Comment:
Ah sorry, I totally forgot: Could you grant the `view_organization` permission to the marketing team and the `change_organization` and `delete_organization` permissions to the service team?

Reply:
Oh, so sorry, the marketing team already has the permission to view them and the service team already the permission to change them...
So please only grant the `delete_organization` permission to the service team :slightly_smiling_face: